tensorflow/tools/pip_package/simple_console_for_windows.py

```diff
@@ -21,8 +21,14 @@
 
 import code
 import sys
+from tensorflow.tools.test import system_info_lib
 
+# print system information..
+def Info(unused_args):
```

Comment:
All the prints when users load TF are usually misleading,  confuses and annoys users.
What would be the exact use case for this?
When exactly would this be shown?